cmd/status.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"log"
+	"time"
+
+	"github.com/allcloud-io/clisso/aws"
+	homedir "github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	RootCmd.AddCommand(cmdStatus)
+	cmdStatus.Flags().StringVarP(
+		&writeToFile, "write-to-file", "w", "",
```

Comment:
Why does `clisso status` need the `-w` flag? This command should only show stuff to the user, not do anything with the credentials, right?